src/styles/shared/variables.css

```diff
@@ -1,3 +1,5 @@
 :root {
-	--header-background-color: rgb(238, 238, 238);
+	--grid-border-color: #dddddd;
```

Comment:
Seems we're going down this path, but I just have to say my opinion... hex is goofy. It's a hack for string encoding and compactness. Graphics programs have always used RGB or HSL, but I guess that's not so relevant since most support hex as well these days (although it is often an extra step/click getting the hex value).

RGBA and HSLA are more widely supported in browsers than hex with alpha. And if you want to programmatically manipulate color values, it's more convenient working with rgb/hsl than hex.